diagnostic.md

```diff
@@ -51,8 +50,7 @@ Write the `curl` request you'd use to delete a single donut, using whatever
 ID you'd like.
 
 ```sh
-# your answer here
-```
+curl --include --request DELETE "http://www.example.com/5"
```

Comment:
The resource `donuts` is missing.

`curl --include --request DELETE http://www.example.com/donuts/1`